apt/templates/unattended_config.jinja

```diff
@@ -1,3 +1,5 @@
+# This file is managed by salt. Manual changes risk being overwritten.
+
```

Comment:
Could we use the method from the `template-formula` instead?

* https://github.com/saltstack-formulas/template-formula/blob/ba911f1b3de0eebe2ed692ddabe2c62a50d9a5b8/TEMPLATE/files/default/example.tmpl.jinja#L1-L4
  - The `source` is then evaluated to the actual location of the template.

Reply:
Excellent. Adding that.

Reply:
Thanks for updating this.